docs/odoo/models/ir.cron.rst

```diff
@@ -0,0 +1,152 @@
+            	<field name="doal">1</field>
+            	<!--<field name="nextcall" ></field>--> 
+            	<field name="model" eval="'model.name '" />
+            	<field name="function" eval="'method_name '" />
+            	<field name="args" eval="" />
+		<!--<field name="priority" eval="5" />--> 
+            </record>
+        </data>
+    </openerp>
+
+The first thing you’ll notice is the data ``noupdate="1"``, this is telling Odoo that all code within this tag shouldn’t be updated when you update your module.
+
+.. code-block:: shell
+
+    <record id="unique_name" model="ir.cron">
+
+The id is an unique identifier for Odoo to know what record is linked to which id. The model called ("ir.cron") is the model specifically made by Odoo for all automated actions. This model contains all automated actions and should always be specified.
+
+.. code-block:: shell
+
+    <field name="name">Name </field>
+
+The next line is the name. 
+
+.. code-block:: shell
+
+    <field name="active" eval="True" />
+
+Boolean value indicating whether the cron job is active or not.
+
+.. code-block:: shell
+
+    <field name="user_id" ref="base.user_root"/>
+
+This user id is referring to a specific user, in most cases this will be base.user_root.
+
+.. code-block:: shell
+
+    <field name="interval_number">1</field>
+
+Number of times the scheduler is to be called based on the "interval_type" 
+
+.. code-block:: shell
+
+    <field name="interval_type">days</field>
+Interval Unit.
+It should be one value for the list: ``'minutes'``, ``'hours'``, ``'days'``, ``'weeks'``, ``'months'``.
+
+.. code-block:: shell
+
+    <field name="numbercall">-1</field>
+
+An integer value specifying how many times the job is executed. A negative value means no limit.
+
+.. code-block:: shell
+
+    <field name="doal">1</field>
+
+A boolean value indicating whether missed occurrences should be executed when the server restarts.
+
+.. code-block:: shell
+
+    <field name="nextcall" ></field>
+
+Next planned execution date for this job.
+
+.. code-block:: shell
+
+    <field name="model" eval="'model.name '" />
+
+The field ``model`` specifies on which model the automated action should be called.
+
+.. code-block:: shell
+
+    <field name="function" eval="'method_name '" />
+
+Name of the method to be called when this job is processed.
+
+.. code-block:: shell
+
+    <field name="args" eval="" />
+
+The arguments to be passed to the method.
+
+.. code-block:: shell
+
+    <field name="priority" eval="5" />
+
+The priority of the job, as an integer: 0 means higher priority, 10 means lower priority.
+
+
+**Defaults.**
+
+
++------------------+-----------------------------------------------------------+
+| Name             | Definition                                                |
++==================+===========================================================+
+| nextcall	   | lambda *a: time.strftime(DEFAULT_SERVER_DATETIME_FORMAT   |
```

Comment:
quote the code:

     +| nextcall	   | ``lambda *a: time.strftime(DEFAULT_SERVER_DATETIME_FORMAT``   |